src/js/utils/dom.js

```diff
@@ -582,6 +582,23 @@ export function findPosition(el) {
   };
 }
 
+/**
+ * Gets element zoom level (reads it from computed style)
+ *
+ * @param  {Element} el
+ *         Element we get the zoom value from
+ *
+ * @return {number} zoom value
+ */
+export function getElementZoomLevel(el) {
+  try {
+    return window.getComputedStyle(el).getPropertyValue('zoom') * 1.0 || 1;
+  } catch (ignore) {
+    log.warn('Can not detect zoom level', el);
```

Comment:
I don't think that we need to log a warning here especially since getPointerPosition is used so often and Firefox doesn't support `zoom`.